postgres-appliance/postgres_backup.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+function wait_for_cluster
+{
+    local SECONDS=${1}
+    [[ -z $1 ]] && SECONDS=0
+    local FINAL_EPOCH=$(date --date "$SECONDS seconds" +%s)
+
+    while true
+    do
+        pg_isready --quiet && return 0
+        [[ $(date +%s) -lt ${FINAL_EPOCH} ]] || break
+        sleep 5
+    done
+
+    echo "ERROR: cluster was not available after $SECONDS seconds" >&2
+    return 1
+}
+
+[[ -z $2 ]] && echo "Usage: $0 WALE_ENV_DIR PGDATA" && exit 1
+
+if [[ ! -z ${INITIAL_BACKUP+1} ]]
```

Comment:
What about good old "x$VAR != x" -- think this is portable?

Side note: line notes made from commit page don't show up in PR...